apps/github-cascading-app/src/cascading/interfaces.ts

```diff
@@ -64,7 +64,7 @@ export type CheckConclusion = 'cancelled' | 'neutral' | 'success' | 'failure' |
 /**
  * Default configuration
  */
-export const DEFAULT_CONFIGURATION: Readonly<CascadingConfiguration> = {
```

Comment:
As discussed, I think the exported const (function, fields, etc...) should keep the type of interface (potentially with `ReadOnly`) and not rely on `satisfies`.
For the following reasons:
- Avoid (future) breaking changes
- Keep the nature of the interface which is to offer a contract with the user